### ISSUE TITLE
Can't install on Windows 7 64bits

### ISSUE THREAD

**Issue by pocesar** [lloyd/node-toobusy#16]
```
C:\Users\Webdev\Inetpub\socketexpress>npm install
npm http GET http://registry.npmjs.org/toobusy
npm http 304 http://registry.npmjs.org/toobusy
npm http GET http://registry.npmjs.org/bindings/1.1.0
npm http 304 http://registry.npmjs.org/bindings/1.1.0

> toobusy@0.2.3 install C:\Users\Webdev\Inetpub\socketexpress\node_modules\toobusy
> node-gyp rebuild


C:\Users\Webdev\Inetpub\socketexpress\node_modules\toobusy>node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node
-gyp.js" rebuild
Traceback (most recent call last):
  File "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\gyp\gyp", line 18, in <module>
    sys.exit(gyp.main(sys.argv[1:]))
  File "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\gyp\pylib\gyp\__init__.py", line 523, in main
    return gyp_main(args)
  File "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\gyp\pylib\gyp\__init__.py", line 499, in gyp_main
    options.circular_check)
  File "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\gyp\pylib\gyp\__init__.py", line 96, in Load
    generator.CalculateVariables(default_variables, params)
  File "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py", line 1703, in CalculateVariables
    generator_flags.get('msvs_version', 'auto'))
  File "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\gyp\pylib\gyp\MSVSVersion.py", line 359, in SelectVisualStudioVersion
    versions = _DetectVisualStudioVersions(version_map[version], 'e' in version)
KeyError: 'vs2012'
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\configure.js:415:16)
gyp ERR! stack     at ChildProcess.EventEmitter.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:757:12)
gyp ERR! System Windows_NT 6.1.7601
gyp ERR! command "node" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\Webdev\Inetpub\socketexpress\node_modules\toobusy
gyp ERR! node -v v0.10.2
gyp ERR! node-gyp -v v0.9.5
gyp ERR! not ok
npm ERR! toobusy@0.2.3 install: `node-gyp rebuild`
npm ERR! `cmd "/c" "node-gyp rebuild"` failed with 1
npm ERR!
npm ERR! Failed at the toobusy@0.2.3 install script.
npm ERR! This is most likely a problem with the toobusy package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls toobusy
npm ERR! There is likely additional logging output above.

npm ERR! System Windows_NT 6.1.7601
npm ERR! command "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "--msvs_version=vs2012"
npm ERR! cwd C:\Users\Webdev\Inetpub\socketexpress
npm ERR! node -v v0.10.2
npm ERR! npm -v 1.2.15
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     C:\Users\Webdev\Inetpub\socketexpress\npm-debug.log
npm ERR! not ok code 0
```

don't know if its a problem with node-gyp or toobusy itself, because other modules are rebuilding fine, toobusy is the only one that's choking. maybe @TooTallNate could chime in as well

**Comment by c4milo** [comment]
@pocesar did you resolve your issue? how did you fix it?

**Comment by pocesar** [comment]
yes, I reinstalled VS 2012, and it appeared to fix it, using SDK 7.1A
